modules/alphamat/tutorials/alphamat_tutorial.markdown

```diff
@@ -7,15 +7,24 @@ This project was part of Google Summer of Code 2019.
 
 *Mentor:* Sunita Nayak
 
-Alphamatting is the problem of extracting the foreground from an image. The extracted foreground can be used for further operations like changing the background in an image.
+Alphamatting is the problem of extracting the foreground with soft boundaries from a background image. The extracted foreground can be used for further operations like changing the background in an image.
 
 Given an input image and its corresponding trimap, we try to extract the foreground from the background. Following is an example:
 
-Input Image: ![](samples/input_images/plant.jpg)
-Input Trimap: ![](samples/trimaps/plant.png)
-Output alpha Matte: ![](samples/output_mattes/plant_result.jpg)
+Input Image: ![](../samples/input_images/plant.jpg)
+Input image should be preferably a RGB image.
```

Comment:
Please check images and formation of the result documentation page [here](http://pullrequest.opencv.org/buildbot/export/pr_contrib/2729/docs/dd/d0e/tutorial_alphamat.html).

See also #2743

Reply:
Updated the image links. The local files image in the same directory as the markdown file are showing fine in GitHub, while the first three ones with path starting from alphamat folder are not. How to check if all image references are working in the final OpenCV doc page?

Reply:
I can also move the samples folder images inside tutorial, if separate samples folder is not necessary. Please let me know.

Reply:
Just refreshed in the [test url ](https://pullrequest.opencv.org/buildbot/export/pr_contrib/2729/docs/dd/d0e/tutorial_alphamat.html) you gave. All images look good now. 👍

Reply:
It is OK to keep images in `samples`, which can be used as sample's input.